chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt

```diff
@@ -391,12 +391,12 @@ open class ConversationFragment() :
     /**
      * This function is for receiving new message and sending new message.
      */
-    private fun addMessageToBottom(message: ChatMessage, uri: Uri? = null) {
+    private fun addMessageToBottom(message: ChatMessage, uri: Uri? = null, orientation: Int? = null) {
```

Comment:
Adding orientation in `ConversationFragment` and `ConversationView` message functions look a bit strange as orientation is for image only, those functions should be generic for adding all message type. Besides that, this PR looks good for me.